src/main/java/org/datadog/jenkins/plugins/datadog/publishers/DatadogQueuePublisher.java

```diff
@@ -49,8 +51,8 @@ of this software and associated documentation files (the "Software"), to deal
     private static final Logger logger = Logger.getLogger(DatadogQueuePublisher.class.getName());
 
     private static final long RECURRENCE_PERIOD = TimeUnit.MINUTES.toMillis(1);
-    private static final Queue queue = Queue.getInstance();
-
+    private final Queue queue = Queue.getInstance();
+    
```

Comment:
I'm not sure about this change. The tests wouldn't recognize items in the queue when it was static. It didn't seem to have an effect on other behavior. If anyone has any insights, let me know!

Reply:
When the field was static the queue instance was fetched the first time this class was loaded. In our tests a jenkins instance is spawned up multiple time leading to `queue` pointing to the wrong queue instance.

I think that's totally fine this way let's keep it like this.